Frontend/src/components/Database/Database.js

```diff
@@ -109,6 +109,17 @@ class Database {
     }
   }
 
+  async countDocs(selectors) {
```

Comment:
since you return a promise without awaiting it this function does not need to be async